.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "crates/generators/minecraft-data"]
 	path = crates/generators/minecraft-data
-	url = git@github.com:PrismarineJS/minecraft-data.git
```

Comment:
Well I also think cargo require you to prefix it with protocol ie `ssh://git@github.com/PrismarineJS/minecraft-data.git`

Reply:
Our Git dependency URLs also use HTTPS, and the issue seems to be fixed now that this PR is merged.

Reply:
Yea, I think we should use the https protocol, just wanted to let you know.